src/events/events-api-client/events-api-client.e2e.ts

```diff
@@ -0,0 +1,37 @@
+import { EventsApiClient } from './events-api-client';
+import { config } from '../../../spec/config';
+import { BugSplatApiClient } from '../../common';
+import { EventStreamEventComment } from '../event-stream/event-stream-event';
+
+describe('CrashApiClient', () => {
+    const email = 'fred@bugsplat.com';
+    const password = 'Flintstone';
+    const database = 'fred';
+    const crashId = 100000;
+    const stackKeyId = 799;
+    let client: EventsApiClient;
+
+    beforeEach(async () => {
+        const bugsplat = new BugSplatApiClient(config.host); 
+        await bugsplat.login(email, password);
+        client = new EventsApiClient(bugsplat);
+    });
+
+    describe('getEventsForCrashId', () => {
+        it('should return 200 for database fred and crashId 100000', async () => {
```

Comment:
Remove the dependency on an event exists on this specific crash/stackKey. The solution is to get a list of crashes and stack keys at test time, pick one, add an event and verify the event was added successfully.

Reply:
We also need to move the database to the config file!